coreservices/partsrelationshipservice/integration-tests/src/test/java/net/catenax/prs/integrationtest/PrsUpdateEventMother.java

```diff
@@ -0,0 +1,127 @@
+//
+// Copyright (c) 2021 Copyright Holder (Catena-X Consortium)
+//
+// See the AUTHORS file(s) distributed with this work for additional
+// information regarding authorship.
+//
+// See the LICENSE file(s) distributed with this work for
+// additional information regarding license terms.
+//
+package net.catenax.prs.integrationtest;
+
+import com.catenax.partsrelationshipservice.dtos.PartAttribute;
+import com.catenax.partsrelationshipservice.dtos.PartLifecycleStage;
+import com.catenax.partsrelationshipservice.dtos.messaging.PartAspectUpdateEvent;
+import com.catenax.partsrelationshipservice.dtos.messaging.PartAttributeUpdateEvent;
+import com.catenax.partsrelationshipservice.dtos.messaging.PartRelationshipUpdateEvent;
+import com.github.javafaker.Faker;
+import net.catenax.prs.testing.DtoMother;
+
+import java.time.Instant;
+import java.util.List;
+
+/**
+ * Object Mother to generate data for integration tests.
+ *
+ * @see <a href="https://martinfowler.com/bliki/ObjectMother.html">
+ * https://martinfowler.com/bliki/ObjectMother.html</a>
+ */
+public class PrsUpdateEventMother {
+
+    /**
+     * JavaFaker instance used to generate random data.
+     */
+    private final transient Faker faker = new Faker();
+
+    /**
+     * Dto mother object that generates test data.
+     */
+    private final DtoMother generate = new DtoMother();
+
+    /**
+     * Samples of the data for PRS Api.
+     */
+    private final PartsTreeApiResponseMother samples = new PartsTreeApiResponseMother();
+
+
+    /**
+     * Generate a {@link PartAttributeUpdateEvent} with sample data.
+     * @return see {@link PartAttributeUpdateEvent}.
+     */
+    public PartAttributeUpdateEvent sampleAttributeUpdateEvent() {
+        return PartAttributeUpdateEvent.builder()
+                .withEffectTime(Instant.now())
+                .withPart(samples.gearwheelpinPartId1())
+                .withName(PartAttribute.PART_TYPE_NAME)
+                .withValue(faker.lorem().word())
+                .build();
+    }
+
+    /**
+     * Generate a {@link PartAttributeUpdateEvent} with sample data.
+     * @param time effect time.
+     * @return see {@link PartAttributeUpdateEvent}.
+     */
+    public PartAttributeUpdateEvent sampleAttributeUpdateEvent(Instant time) {
```

Comment:
nit: just for relevant name. Feel free to ignore. thanks
```suggestion
    public PartAttributeUpdateEvent sampleAttributeUpdateEvent(Instant effectTime) {
```
(in some other methods as well in this class)